image-layout.md

```diff
@@ -2,6 +2,8 @@
 
 The OCI Image Layout is a slash separated layout of OCI content-addressable blobs and [location-addressable](https://en.wikipedia.org/wiki/Content-addressable_storage#Content-addressed_vs._location-addressed) references (refs).
 This layout MAY be used in a variety of different transport mechanisms: archive formats (e.g. tar, zip), shared filesystem environments (e.g. nfs), or networked file fetching (e.g. http, ftp, rsync).
+This section defines none of media type for image layout.
```

Comment:
is this necessary? its inclusion just confuses me. a layout as described here couldn't really have a media type